PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -1073,6 +1073,29 @@ GetRegionInfo (
   return EFI_SUCCESS;
 }
 
+/**
+  End firmware update.
+
+  This function will clear firmware update trigger and end firmware update.
+
+  @retval  EFI_SUCCESS        Update successfully.
+  @retval  others             Error happened during end firmware update.
+
+**/
+EFI_STATUS
+EFIAPI
+EndFirmwareUpdate (
+  VOID
+  )
+{
+  EFI_STATUS  Status;
+
+  Status = ClearFwUpdateTrigger ();
+
+  Status = PlatformEndFirmwareUpdate ();
+  return Status;

```

Comment:
There are two status above.   Either of the failure should result in the final failure.

Reply:
From the original design, clear fwu flag is part of the EndFirmwareUpdate from platform.
do we have the case for the core (FirmwareUpdate driver) to only call one of them? want to understand why we need have API ClearFwUpdateTrigger () for the core driver.

Reply:
ClearFwUpdateTrigger() will need to be done at the beginning of the FWU flow as well after the sticky register is detected and translated into the state machine.  So it needs to be called from core side.

Reply:
Since ClearFwUpdateTrigger does not potentially return an error, changed return from status to VOID.